app/views/shared/_cookie_banner.html.erb

```diff
@@ -3,21 +3,11 @@
 
   <div id="cookie-banner-message" class="govuk-width-container"
     data-cookie-acceptance-target="banner">
-
-    <p>
-        Welcome to the new Get an Adviser service. We invite you to try the
-        latest beta version of the site and help us improve the services we offer.
-    </p>
+    <p><strong>Tell us whether you accept cookies</strong></p>
     <p>
-        To use the new website, we need you to agree to our cookie policy. This
-        means we can use your data to understand how to improve the website
-        for all of our users.
+    We use <%= link_to "cookies to collect information", cookies_path %> about how you use this site. 
+    We use this information to make the website work as well as possible and improve our services.
```

Comment:
This reads a bit funny to me: "work as well as possible" -- are they not really just the same thing? "to make the website work and improve our services" reads better to me?

Reply:
I've used this for inspiration, so was hoping to keep as is


![image](https://user-images.githubusercontent.com/35870975/113733452-63781c00-96f2-11eb-9df1-6db7051adab2.png)

Reply:
Fair enough 👍